indent/verilog_systemverilog.vim

```diff
@@ -103,7 +103,7 @@ function GetVerilog_SystemVerilogIndent()
     \ last_line =~ '^\s*\<\(always\|always_comb\|always_ff\|always_latch\)\>' ||
     \ last_line =~ '^\s*\<\(initial\|specify\|fork\|final\)\>' ||
     \ last_line =~ '^\s*\(\w\+\s*:\s*\)\?\<\(assert\|assume\|cover\)\>'
-    if last_line !~ '\(;\|\<end\>\)\s*' . vlog_comment . '*$' ||
+    if last_line !~ '\(;\|\<end\>\|`\h\w*\((.*)\)\?\)\s*' . vlog_comment . '*$' ||
```

Comment:
Could you move the check to the following line as:

    \ last_line !~ '`\h\w*\((.*)\)\?\s*' . vlog_comment . '*$' ||

They are searching for different types of statements, so it makes sense to have them separate.